thirdparty/vecmem/CMakeLists.txt

```diff
@@ -16,11 +16,15 @@ message( STATUS "Building VecMem as part of the Acts project" )
 # Declare where to get VecMem from.
 set( ACTS_VECMEM_GIT_REPOSITORY "https://github.com/acts-project/vecmem.git"
    CACHE STRING "Git repository to take VecMem from" )
-set( ACTS_VECMEM_GIT_TAG "v0.4.0" CACHE STRING "Version of VecMem to build" )
+set( ACTS_VECMEM_GIT_TAG "v1.4.0" CACHE STRING "Version of VecMem to build" )
 mark_as_advanced( ACTS_VECMEM_GIT_REPOSITORY ACTS_VECMEM_GIT_TAG )
 FetchContent_Declare( VecMem
    GIT_REPOSITORY "${ACTS_VECMEM_GIT_REPOSITORY}"
    GIT_TAG "${ACTS_VECMEM_GIT_TAG}" )
 
+# Options used in the build of VecMem.
+set( VECMEM_BUILD_TESTING FALSE CACHE BOOL
+   "Turn off the build of the VecMem unit tests" )
```

Comment:
It is the default (currently), but I wonder if it may make sense to set `VECMEM_BUILD_BENCHMARKING` to `FALSE` explicitly here as well. :thinking: